asposeimagingcloud/models/bmp_properties.py

```diff
@@ -0,0 +1,122 @@
+# coding: utf-8
+# -----------------------------------------------------------------------------
+# <copyright company="Aspose" file="BmpProperties.py">
```

Comment:
@s-zubov Please, fix the model templates for copyright file names to correspond with actual file names.

Reply:
There is no such variable that contains the file name. I've created a console script that fixes this issue after models are generated (at [codegen repo](https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-codegen/pull/25/commits/e20abbb255d3af41c671f9cfe004b7537c30c7ca))